### PR TITLE
Added check if pr got 'ready-for-tests' label to save test results capacity

### PR DESCRIPTION
**Pull request by niemam29**
As we already used 3/4 of test results capacity i changed a bit the way of executing tests, right now, unless there is no label 'ready-for-tests' label on pr the cypress smoke tests will not be executed.

cach2

This is temporary solution as i am researching possibility to switch to sorry-cypress

### CODE REVIEW

**Review by mix1o — APPROVED** [review]

**Review by mix1o — CHANGES_REQUESTED** [review]
test

**Review by mwiniarsk — APPROVED** [review]

**Review by mix1o — APPROVED** [review]

**Comment by wixzi** [comment]
@niemam29 Nice idea to check labels and run cypress. 🍠 

Do you still want to dump the Github context. Did you just done for debugging?
Also can't we add a label and rerun the action instead of creating a commit each time for Synpress?. 

If possible I suggest you do a label check for synpress also. And then rerun the action in the actions page instead of creating an empty commit in each PR.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,28 +19,69 @@ concurrency:
...
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo "$GITHUB_CONTEXT"
+
       - name: Generate unique ID 💎
         id: uuid
-        run: echo "::set-output name=value::sha-$GITHUB_SHA-time-$(date +"%s")"
+        run: |
+          echo "::set-output name=value::sha-$GITHUB_SHA-time-$(date +"%s")"
+          echo "sha-$GITHUB_SHA-time-$(date +"%s")"
```
Comment (wixzi) [inline .github/workflows/ci.yml:47]: Why we need this UUID?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -19,28 +19,69 @@ concurrency:
...
       - name: ⬇️ ・Checkout repo
         uses: actions/checkout@v3
         with:
           fetch-depth: '50'
 
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo "$GITHUB_CONTEXT"
```
Comment (wixzi) [inline .github/workflows/ci.yml:41]: I think we can remove this.

**Comment by wixzi** [comment]
`  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
`

**Review by MilanVojnovic95 — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (niemam29) [reply .github/workflows/ci.yml:47]: Cypress dashboard uses commit sha as a way to identify single test run, in order to be able to run tests using rerun button i added timestamp to this sha

File: .github/workflows/ci.yml (hunk 2 above)
Comment (niemam29) [reply .github/workflows/ci.yml:41]: I will leave it for a few runs (mostly when merging to develop) to see if everything works good

**Comment by niemam29** [comment]
> ```
> 
> # Allows you to run this workflow manually from the Actions tab
> workflow_dispatch:
> ```

I will exclude tests back to CT pipe and then create a whole test runner using this dispatch

**Pull request merged**